fetch.js

```diff
@@ -0,0 +1,70 @@
+var Bluebird  = require('bluebird');
+var _defaults = require('lodash/object/defaults');
+
+var EmberCliDeployError = require('./errors/ember-cli-deploy-error');
+
+var redis = require('then-redis');
```

Comment:
Consider naming this with class name conventions (`ThenRedis`) and update related rewires. I think that would make the tests read better.